ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -86,6 +85,13 @@
       regexFr: Regexes.startsUsing({ id: '4BAB', source: 'Ramuh', capture: false }),
       regexJa: Regexes.startsUsing({ id: '4BAB', source: 'ラムウ', capture: false }),
       regexKo: Regexes.startsUsing({ id: '4BAB', source: '라무', capture: false }),
+      preRun: function(data) {
+        data.fourteenCount = data.fourteenCount || 0;
+        data.fourteenCount++;
+      },
+      condition: function(data) {
```

Comment:
Confusingly, I think preRun only happens if condition is met, and runs after the condition.  So maybe this needs to be: `return !data.fourteenCount || data.fourteenCount < 2`.  This makes me wonder if I should write a test that all the fields have to be in the order they run, for clarity.